tests/snuba/api/endpoints/test_organization_events_span_metrics.py

```diff
@@ -1438,6 +1438,53 @@ def test_count_op(self):
             {"count_op(queue.submit.celery)": 1, "count_op(queue.task.celery)": 1},
         ]
 
+    def test_frames_metrics(self):
```

Comment:
your tests pass because the store_span_metric function actually storing your span metrics on the distributions table. this is because you're not explicitly passing the [metric](https://github.com/getsentry/sentry/blob/master/src/sentry/testutils/cases.py#L2178) arg and that's what is used to route the value to the correct table, so it's getting stored on the `distributions` table (since default is span.self_time`